src/components/swap/SwapButtons/SwapButton.tsx

```diff
@@ -48,25 +49,36 @@ const StyledPlataformText = styled(Text)`
 
 interface SwapButtonProps {
   platformName?: string
-  swapInputError?: string
+  swapInputError?: number
   priceImpactSeverity: number
   isExpertMode: boolean
+  amountInCurrencySymbol?: string
 }
 
 export const SwapButton = ({
   platformName,
   swapInputError,
   priceImpactSeverity,
   isExpertMode,
+  amountInCurrencySymbol,
   ...rest
 }: SwapButtonProps & ButtonProps) => {
   const { t } = useTranslation()
 
+  const SWAP_INPUT_ERRORS_MESSAGE = {
+    [SWAP_INPUT_ERRORS.CONNECT_WALLET]: 'Connect wallet',
+    [SWAP_INPUT_ERRORS.ENTER_AMOUNT]: 'Enter amount',
+    [SWAP_INPUT_ERRORS.SELECT_TOKEN]: 'Select token',
+    [SWAP_INPUT_ERRORS.ENTER_RECIPIENT]: 'Enter recipient',
+    [SWAP_INPUT_ERRORS.INVALID_RECIPIENT]: 'Invalid recipient',
+    [SWAP_INPUT_ERRORS.INSUFFICIENT_BALANCE]: `Insufficient ${amountInCurrencySymbol} balance`,
+  }
```

Comment:
Nice structure, better to reason about things :)

Reply:
Will add translations soon!